include/suit_types.h

```diff
@@ -15,11 +15,14 @@
 
 #define SUIT_MAX_NUM_SIGNERS 2  ///! The maximum number of signers.
 #define SUIT_MAX_NUM_COMPONENT_ID_PARTS 5  ///! The maximum number of bytestrings in a component ID.
-#define SUIT_MAX_NUM_COMPONENTS 4  ///! The maximum number of components referenced in the manifest.
+#define SUIT_MAX_NUM_COMPONENTS 6  ///! The maximum number of components referenced in the manifest.
```

Comment:
This definition is unclear in a sense that it describes Max number of components per manifest, but naming may suggest it refers to max number of components in entire SUIT tree

Reply:
Postponed due to code freeze (requires changes in sdk-ic-next)